### DISCUSSION TITLE
How to build the same situation on 3d model?

### DISCUSSION THREAD

**Discussion by PenuverWoo** [ricktu288/ray-optics#139] [Ideas]
i d like to analyze the reflection on 3d models, like diamond, any suggestions for how to build a simulation.

**Comment by RainerHeintzmann** [comment]
It would be interesting to have a 3D tool, but as a step towards it, one could also construct it by showing both, an XZ and a YZ (projection) view such that both views can be manipulated. Of course the rays would need to be traced in 3D.

**Comment by ricktu288** [comment]
Note that https://www.optico.app/ (which may be considered a very distant fork of this project) has the 3D spot diagram feature, even if the editor UI is completely in 2D.
